pkg/secrethub/crypto.go

```diff
@@ -63,8 +63,8 @@ func (c *Client) encryptSecretFor(secret *api.Secret, accounts ...*api.Account)
 			}
 
 			encryptedKeys[keyIndex] = api.SecretKeyMemberRequest{
-				AccountID:    account.AccountID,
-				SecretKeyID:  decryptedKey.SecretKeyID,
+				AccountID:    &account.AccountID,
```

Comment:
I have only changed to values instead of pointers for responses (of http- and regular client). For http requests, I have left the pointers, so that we are able to check whether the value was set or left to the zero value. In this specific case, we do so in `api.SecretKeyMemberRequest.Validate`.

Reply:
Alright, but is that really what we should check? If it is a required field, can we check if it is the zero value instead?